lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -191,50 +97,12 @@ private static function translate_theme_json_chunk( array $array_to_translate, $
 	 * @return array Returns the modified $theme_json_structure.
 	 */
 	private static function translate( $theme_json, $domain = 'default' ) {
-		$fields = self::get_fields_to_translate();
-		foreach ( $fields as $field ) {
-			$path    = $field['path'];
-			$key     = $field['key'];
-			$context = $field['context'];
-
-			/*
-			 * We need to process the paths that include '*' separately.
-			 * One example of such a path would be:
-			 * [ 'settings', 'blocks', '*', 'color', 'palette' ]
-			 */
-			$nodes_to_iterate = array_keys( $path, '*', true );
-			if ( ! empty( $nodes_to_iterate ) ) {
-				/*
-				 * At the moment, we only need to support one '*' in the path, so take it directly.
-				 * - base will be [ 'settings', 'blocks' ]
-				 * - data will be [ 'color', 'palette' ]
-				 */
-				$base_path = array_slice( $path, 0, $nodes_to_iterate[0] );
-				$data_path = array_slice( $path, $nodes_to_iterate[0] + 1 );
-				$base_tree = _wp_array_get( $theme_json, $base_path, array() );
-				foreach ( $base_tree as $node_name => $node_data ) {
-					$array_to_translate = _wp_array_get( $node_data, $data_path, null );
-					if ( is_null( $array_to_translate ) ) {
-						continue;
-					}
-
-					// Whole path will be [ 'settings', 'blocks', 'core/paragraph', 'color', 'palette' ].
-					$whole_path       = array_merge( $base_path, array( $node_name ), $data_path );
-					$translated_array = self::translate_theme_json_chunk( $array_to_translate, $key, $context, $domain );
-					gutenberg_experimental_set( $theme_json, $whole_path, $translated_array );
-				}
-			} else {
-				$array_to_translate = _wp_array_get( $theme_json, $path, null );
-				if ( is_null( $array_to_translate ) ) {
-					continue;
-				}
-
-				$translated_array = self::translate_theme_json_chunk( $array_to_translate, $key, $context, $domain );
-				gutenberg_experimental_set( $theme_json, $path, $translated_array );
-			}
+		if ( null === self::$i18n_schema ) {
+			$i18n_schema       = wp_json_file_decode( __DIR__ . '/theme-i18n.json' );
```

Comment:
I think this should be `gutenberg_json_file_decode`, or we should rename the function. It is causing a fatal error, and I'm not able to access the site editor.